addons/mrp/wizard/mrp_product_produce.py

```diff
@@ -130,6 +134,16 @@ def _record_production(self):
                 'date_start': datetime.now(),
             })
 
+    def _check_strict_consumption(self, lines_to_check):
```

Comment:
lines_to_check is useless, self.workorder_line_ids will always contain everything...